image/golem_blender_app/golem_blender_app/verifier_tools/image_metrics_calculator.py

```diff
@@ -60,7 +60,9 @@ def calculate_metrics(
         print("There were errors %r" % e, file=sys.stderr)
         compare_metrics['Label'] = VERIFICATION_FAIL
     providers_result_crop.save(
-        _generate_path_for_providers_result_crop(reference_crop_path)
+        os.path.join(
```

Comment:
`Path` over `os.path`?

Reply:
`Path` is only used once in a constant definition, apart from that `os.path` is used everywhere else. I didn't want to create new problems to solve.